openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -84,16 +84,14 @@
     private static final String OPENAPI_DEFAULTED_STATIC_FILE_LOG_MESSAGE_FORMAT = "Using default OpenAPI static file %s";
 
     private final String webContext;
-    private final Builder builder;
-    private final OpenApiConfig openAPIConfig;
 
-    private OpenAPI model = null;
+    private final OpenAPI model;
     private final ConcurrentMap<Format, String> cachedDocuments = new ConcurrentHashMap<>();
 
     private OpenAPISupport(final Builder builder) {
         webContext = builder.webContext();
-        this.builder = builder;
-        this.openAPIConfig = builder.openAPIConfig();
+        model = prepareModel(builder.openAPIConfig(), builder.indexView(),
+                    builder.staticFile());
```

Comment:
Just a nit: this looks like it would fit just fine on the previous line.

Reply:
Fixed.